elasticassignment/ageofempires/models/civilizations.py

```diff
@@ -0,0 +1,52 @@
+from elasticsearch.helpers import bulk
+from django.conf import settings
+from ageofempires.Elastic_search.elastic_client import elastic_client
+
+class Civilizations():
+
+    elastic_client = elastic_client()
+
+    def __get_settings():
+        return {
+            "index":{
+                    "number_of_shards" : settings.NUMBER_OF_SHARDS,
+                    "number_of_replicas" : settings.NUMBER_OF_REPLICAS
+            }
+        }
+
+    def __get_mappings():
+        return {
+            'properties': {
+                    "army" : {"type" : "text"},
+                    "civilization_bonus" : {"type" : "text"},
+                    "id" : {"type" : "integer"},
+                    "name" : {"type" : "text"},
+                    "team_bonus" : {"type" : "text"},
+                    "unique_tech" : {"type" : "text"},
+                    "unique_unit" : {"type" : "text"}
+            }
+        }
+
+    def make_index():
+        request_body = {
+            "settings" : Civilizations.__get_settings(),
+            'mappings' : Civilizations.__get_mappings()
+        }
+        print("creating 'civilization' index...")
+        es  = elastic_client.make_connection()
+        es.indices.create(index = 'civilization', body = request_body)
+
+    def populate_index(url:str):
```

Comment:
- please redefine as: `bulk_index(data:list)`
- move data processing and batching out of this function (will move to the `import_data` command)

Reply:
- model classes will also need `get_objects` or similar function.